build.gradle

```diff
@@ -87,6 +88,10 @@ dependencies {
   testImplementation 'com.github.stefanbirkner:system-rules:1.18.0'
   testImplementation "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
   testImplementation "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+
+  integrationTestImplementation gradleTestKit()
+  integrationTestImplementation 'junit:junit:4.12'
```

Comment:
@vanniktech should we extract common dependencies versions (`junit` and `assertj`) into constants or local variables inside `dependencies` block?

Reply:
Yes just do it similar to `$kotlinVersion` so `junitVersion = "4.12"` & `assertj3Version = "3.12.1"`